src/views/Token/index.tsx

```diff
@@ -1,9 +1,11 @@
 // External
 import React, { useState, useEffect } from 'react'
 import dayjs from 'dayjs'
+import { useWeb3React } from '@web3-react/core'
 import { BigNumber } from 'ethers'
 import { useTranslation } from 'react-i18next'
 import { useDispatch } from 'react-redux'
+import _ from 'lodash'
```

Comment:
Lodash is something we should really try best not to use in the frontend. It has a huge package size that really slows down page loads.
If we have a legitimate use for it then its fine but I think what is being done here can be done with vanilla es6 easily enough.

Reply:
Good point. Import only the functions you need to use via the [`es` package](https://www.npmjs.com/package/lodash-es)

Reply:
I didn't know about lodash slowing down pages. I liked its methods however as it is a problem, I'll remove it and use classic ES6 functions instead.